defaults.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     '@apostrophecms/util': {},
     '@apostrophecms/task': {},
     '@apostrophecms/schema': {},
-    '@apostrophecms/attachment': {},
+    '@apostrophecms/uploadfs': {},
```

Comment:
Is there sense in making this '@apostrophecms/uploader'? Might people want to use an alternative or something like that? Or is uploadfs so baked in that we are one?

Reply:
This module is similar to `@apostrophecms/express` and `@apostrophecms/db` in that the module is not interesting to outside code, the actual service is. I don't think the name should change to suggest it's anything more than "the thing that fires up uploadfs and then you have uploadfs."

Reply:
The issue I have is that it is something people will configure. If you want to add your own image size, you  have to configure it. Express and "database" are relatively generic or well-known. uploadfs is a really deep cut package with a funny name. If we're asking people to configure it to do a pretty normal thing, let's give them a pretty recognizable label to reference.